pkg/filter/filter.go

```diff
@@ -107,30 +123,3 @@ func filterByPattern(entity string, include, exclude []regex.Regex) error {
 
 	return nil
 }
-
-// filterFileEntity determines if a heartbeat of type file should be skipped, by verifying
-// the existence of the passed in filepath, and optionally by checking if a
-// wakatime project file can be detected in the filepath directory tree.
-// Returns an error to signal to the caller to skip the heartbeat.
-func filterFileEntity(heartbeat heartbeat.Heartbeat, config Config) error {
```

Comment:
I wouldn't remove this func beause it has a meaning and is well described. Maybe check if is remote in it and describe that in the func comments?